base/testfiles/github-1606.lvt

```diff
@@ -0,0 +1,18 @@
+\begin{filecontents*}[overwrite]{testpackage.sty}
+\ProvidesExplPackage{testpackage}{2025-02-04}{}{}
+\keys_define:nn { testpackage }
+  { testoption .code:n = { \def \testmacro { seen } } }
```

Comment:
this doesn't test the load-only code, and if I add `testoption .usage=load` then the duplicate load still produces a warning:

```

LaTeX Warning: Package "testpackage" has already been loaded: ignoring
(LaTeX)        load-time option "testoption".
```

Reply:
Thanks - fixed